### PR TITLE
DOT-3178: StoreAdmin > API > Product Reviews

### PR DESCRIPTION
**Pull request by 5baddi**
## JIRA Ticket

[DOT-3178](https://nextmediama.atlassian.net/browse/DOT-3178)

### CODE REVIEW

File: docs/products/reviews/get.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "ratings": 3,
+    "first_name": "Mohamed",
+    "last_name": "",
+    "content": "",
+    "images_urls": [],
+    "created_at": "2021-04-21T12:08:46+00:00"
+}
+```
+
+[404] Review not found
```
Comment (mikhalkov5) [inline docs/products/reviews/get.md:27]: if you can add response here

File: docs/products/reviews/delete.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+```json
+{
+    "message": "Product review deleted successfully",
+    "http_code": 200,
+    "code": "DELETED",
+    "type": "success"
+}
+```
+
+[404] Review not found
```
Comment (mikhalkov5) [inline docs/products/reviews/delete.md:24]: same

**Review by mikhalkov5 — APPROVED** [review]
LGME!, small comment left

**Comment by 5baddi** [comment]
Looks good

**Comment by 5baddi** [comment]
Looks good

**Pull request merged**